recipes/mo-catnip/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "mo-catnip" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 65d1c726662069114d4bba6226d621fc0a8d1250b2210ed23763573ab8e87d8c
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+
+  run:
+    - python >=3.6
+    - iris
+    - numpy
+    - six
+    - dask
+    - matplotlib
+
+test:
+  imports:
+    - lib/catnip
```

Comment:
This line should be whatever normal usage of the CATNIP library is. I would assume it's just `import catnip` so this line should just be `- catnip` but I'm not a user of this.